pkg/minikube/cluster/cluster.go

```diff
@@ -39,11 +39,13 @@ import (
 	"github.com/minishift/minishift/pkg/minishift/registration"
 	minishiftUtil "github.com/minishift/minishift/pkg/minishift/util"
 	"github.com/minishift/minishift/pkg/util"
+	"github.com/pkg/errors"
```

Comment:
Not sure why in some cases the standard errors package is added like this. Happened to me too.